ui/src/pages/backoffice/components/OrderList/OrderListEntry.js

```diff
@@ -30,8 +30,14 @@ export default class OrderListEntry extends Component {
             {order.metadata.vendor.name}
           </Link>
         </Item.Description>
+        {order.metadata.grand_total.currency !== order.metadata.grand_total_main_currency.currency && (
```

Comment:
Just to understand, is this meant to be `if...else` conditional rendering or just render the below label in case the condition is satisfied? I am asking because the label for both `<Item.Description>` elements is `grand total`

Reply:
@zzacharo this is conditional display, show it only if the currencies are different. Indeed the label is the same, but the currencies differentiate them and looks like this
![Screenshot 2020-01-07 13 30 09](https://user-images.githubusercontent.com/230805/71895630-dbebd800-3151-11ea-81cd-1ba3085a0e3b.png)

Reply:
Isn't it better to just put the default currency in parenthesis in case it's different than the order currency? Like `70.91 USD (72 CHF)`?

Reply:
Now that I see it, please do not display "grand" total to the user: this is more a technical term, I doubt you ever saw it in an e-commerce website...

Reply:
I think I would put them the other way arround

`grand total 80 CHF (75 EUR)`

Reply:
I changed it in order to reflect the sort option from the dropdown, I will change the dropdown too, to display Total instead of Grand Total